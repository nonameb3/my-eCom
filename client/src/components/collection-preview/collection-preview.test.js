import React from "react";
import { shallow } from "enzyme";

import CollectionPreview from "./collection-preview.component";
import CollectionItem from "../collection-item/collection-item.component";

let wrapper;
const mockItems = [{ id: 1 }, { id: 2 }, { id: 3 }];
const mockProps = { items: mockItems, title: "test" };

describe("collection-preview component", () => {
  beforeEach(() => {
    wrapper = shallow(<CollectionPreview {...mockProps} />);
  });

  it("should render collection-preview component", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("should render collection-item as expect", () => {
    expect(wrapper.find(CollectionItem).length).toBe(mockItems.length);
  });
});
