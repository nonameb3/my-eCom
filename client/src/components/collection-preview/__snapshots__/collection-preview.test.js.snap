// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`collection-preview component should render collection-preview component 1`] = `ShallowWrapper {}`;
